os/board/imxrt1050-evk/src/imxrt_boot.c

```diff
@@ -161,14 +161,12 @@ void imxrt_spi_initialize(void)
 	struct spi_dev_s *spi;
 	spi = up_spiinitialize(1);
 
-	char path[10];
-	if (spi != NULL) {
-		snprintf(path, sizeof(path), "/dev/spi%d", 1);
-		if (spi_uioregister(path, spi) < 0) {
-			lldbg("Failed to register SPI%d\n", 1);
-		}
+#ifdef CONFIG_SPI_USERIO
+	if (spi_uioregister(1, spi) < 0) {
```

Comment:
Why don't you use a defined name as a bus number?

Reply:
Enabling SPI in NXP is on progress. so It'll be refactored and modified when SPI is stable.